pygmt/tests/test_info.py

```diff
@@ -33,7 +33,38 @@ def test_info_dataframe():
     table = pd.read_csv(POINTS_DATA, sep=" ", header=None)
     output = info(table=table)
     expected_output = (
-        "<matrix memory>: N = 20 <11.5309/61.7074> <-2.9289/7.8648> <0.1412/0.9338>\n"
+        "<vector memory>: N = 20 <11.5309/61.7074> <-2.9289/7.8648> <0.1412/0.9338>\n"
+    )
+    assert output == expected_output
+
+
+def test_info_pandas_dataframe_time_column():
+    "Make sure info works on pandas.DataFrame inputs with a time column"
+    table = pd.DataFrame(
+        data={
+            "z": [10, 13, 12, 15, 14],
+            "time": pd.date_range(start="2020-01-01", periods=5),
+        }
+    )
+    output = info(table=table)
+    expected_output = (
+        "<vector memory>: N = 5 <10/15> <2020-01-01T00:00:00/2020-01-05T00:00:00>\n"
+    )
+    assert output == expected_output
```

Comment:
Note that this test currently fails, giving what looks like a UNIX timestamp rather than a ISO datetime output:

```
E       AssertionError: assert '<vector memo...1578182400>\n' == '<vector memo...5T00:00:00>\n'
E         - <vector memory>: N = 5 <10/15> <2020-01-01T00:00:00/2020-01-05T00:00:00>
E         + <vector memory>: N = 5 <10/15> <1577836800/1578182400>
```

Will need to investigate, but any idea why this is happening?

Reply:
GMT_Put_Vector started to accept datetime strings in PR https://github.com/GenericMappingTools/gmt/pull/3396. In this implementation, all datetime strings are converted to `double` internally.

Looks like a GMT API bug to me.

Reply:
So the inputs are read correctly here, we can pass in datetime types using the `GMT_DATETIME` enum. The output is also the correct UNIX timestamp range, but I doubt it's something users would expect.

I suppose we can `xfail` this for 6.1.1, but would prefer to have this reported and/or fixed in GMT master first (i.e. for 6.2.0). A workaround might be to use `np.datetime64(1577836800, 's')` to get `numpy.datetime64('2020-01-01T00:00:00')`, but this isn't a nice solution.

Reply:
> I suppose we can `xfail` this for 6.1.1,

Yes to me.